components/SpotForm.js

```diff
@@ -48,19 +48,17 @@ export default function SpotForm({ addSpots }) {
   const onSubmitHandler = (event) => {
     event.preventDefault();
     const newSpot = {
-      id: formData.id,
-      Spotname: formData.Spotname,
-      Address: formData.Address,
-      Tasks: formData.Tasks,
-      Equipment: formData.Equipment,
+      id: uuidv4(),
+      name: formData.Name,
+      address: formData.Address,
+      tasks: formData.Tasks,
+      equipment: formData.Equipment,
     };
-    const existingSpots = JSON.parse(localStorage.getItem("spots")) || [];
-    const updatedSpots = [...existingSpots, newSpot];
-    localStorage.setItem("spots", JSON.stringify(updatedSpots));
-    addSpots(newSpot);
+
+    addSpot(newSpot);
     setFormData({
       id: Date.now(),
-      Spotname: "",
+      Name: "",
       Address: "",
       Tasks: [],
       Equipment: [],
```

Comment:
You renamed the object properties –> Good!
Then you should also rename the form fields (lowercase).